domains/davechbga.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "davechbga",
+        "email": "davechbga@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "davechbga.vercel.app"
```

Comment:
This should be an actual record type like CNAME. See docs: https://docs.is-a.dev/guides/vercel/